Basic-Car-Maintenance/Shared/Settings/ViewModels/ChooseAppIconViewModel.swift

```diff
@@ -0,0 +1,40 @@
+import UIKit
```

Comment:
You deleted the header comments, as copied from the `ChooseAppIconView`, go ahead and keep them there, cause they're on all the other files right now
```swift
//
//  ChooseAppIconView.swift
//  Basic-Car-Maintenance
//
//  Created by Daniel Lyons on 10/11/23.
```

Reply:
👍🏼